cime_config/testdefs/testlist_clm.xml

```diff
@@ -4279,6 +4282,24 @@
       <option name="comment">Initialization test for mpasa3p75 default layout</option>
     </options>
   </test>
+  <test name="SMS_Ln1_PL" grid="mpasa3p75_mpasa3p75_mt13" compset="I2000Clm45Sp" testmods="clm-for_testing_fastsetup_bypassrun--clm-mpasa3p75">
+    <machines>
+      <machine name="derecho" compiler="intel" category="uhr_decomp_init"/>
+    </machines>
+    <options>
+      <option name="wallclock">1:00:00</option>
+      <option name="comment">Initialization test for mpasa3p75 "Large" layout</option>
+    </options>
+  </test>
+  <test name="SMS_Ln1_PLhm" grid="mpasa3p75_mpasa3p75_mt13" compset="I2000Clm45Sp" testmods="clm-for_testing_fastsetup_bypassrun--clm-mpasa3p75">
```

Comment:
Does it make sense to add new tests with Clm45 compsets (here and above) when we intend to stop maintaining Clm45 beyond a bare minimum?

Reply:
Changing this would likely register as a change in answers, so I figure I'll leave it for now. But, I'll look into changing it for all the tests....